src/icons/service-serverless.svg

```diff
@@ -0,0 +1 @@
+<svg viewBox="0 0 76 124" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="m7.591 123.963 61.944-56.105-4.6-2.888-52.506 47.52v-11.24l58.317-52.752V59.82l-5.81 5.149 4.6 2.888 6.204-5.51V40.46l-13.582 1.88 12.176-20.386V2.723L71.635.279.331 61.716v15.037l22.875-3.863-15.62 26.42v24.653h.005zM69.346 12.539v7.92l-13.071 22.69-6.654.92 19.725-31.53zm-64.221 58.2v-8.12l27.836-3.796-5.844 8.43L5.125 70.74z" fill="#FD5750"/></svg>
```

Comment:
These are general change we make with svgs.
remove fill="none" in svg tag
change fill="currentColor" in path tag
Please make the change unless otherwise required

Reply:
Could not find clean way to use icon with currentColor.